DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/highlightQueryResultsBasics/metadata.json

```diff
@@ -0,0 +1,239 @@
+{
+    "Slug": "highlight-query-results-basics",
+    "SourceFileName": "HighlightQueryResultsBasics.java",
+    "Title": "Highlight Query Results Basics",
+    "DescriptionLines": [
+        "When making a Full Text Search query,<br> in addition to getting the documents that contain the searched terms in the results,",
+        "you can **request to get a list of text fragments highlighting those terms**.",
+        "",
+        "The highlighted terms show exactly why the resulting documents have matched.",
+        "",
+        "The following can be customized:",
+        "<p>* Fragments length<br>",
+        "* Fragments count<br>",
+        "* Tags marking the terms within the fragments<br>",
+        " (see the Customized Demo )</p>"
+    ],
+
+    "Assets": [
+        {
+            "Title": "FTS - Static Index Demo",
+            "Type": "demo",
+            "Url": "/java/text-search/fts-with-static-index-single-field"
```

Comment:
pls return all spaces to how it was, 
we talked about that in the very beginning... 
the current convention for all metadata files in the demo is:


![image](https://user-images.githubusercontent.com/17376974/168488176-4a5f4307-85b9-4adf-a6b2-0a0433cf98a6.png)

Reply:
done